.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
 .DS_Store
+Gemfile.lock
```

Comment:
Facepalm, was doing the same in PR: https://github.com/theforeman/community-templates/pull/486/files

Removed.